test/rate_weet.spec.js

```diff
@@ -0,0 +1,98 @@
+const jwt = require('jwt-simple');
+const supertest = require('supertest');
+const { factory } = require('factory-girl');
+const { User, Weet, Rating } = require('../app/models');
+const app = require('../app');
+const { mockUserVerified, mockUser, mockUserDeveloper } = require('./mocks/mockUsers.js');
+const { VALIDATION_ERROR, AUTHORIZATION_ERROR } = require('../app/errors');
+
+const verify = jest.spyOn(jwt, 'decode');
+const request = supertest(app);
+factory.define('user', User, {});
+factory.define('weet', Weet, { content: 'loremIpsum', userId: factory.assoc('user', 'id') });
+factory.define('rating', Rating, {
+  ratingUserId: factory.assoc('user', 'id'),
+  weetId: factory.assoc('weet', 'id'),
+  score: 1
+});
```

Comment:
Do any factory definition in a dedicated file, not inside a test suite (.spect.js file)